packages/building-webpack/src/modern-web-webpack-plugin.js

```diff
@@ -54,6 +54,11 @@ const legacyError = error(
   'You are running webpack-dev-server with a legacy build, run the dev server with the --legacy flag for this browser or run a production build.',
 );
 
+function pathJoin(...args) {
```

Comment:
I suggest naming this browserPathJoin or something like that. Also, why do we need to call apply here?

Reply:
`path.join takes` a list of n argument. can you `forward` n arguments without using apply?

Reply:
Yeah I think you can just do `path.join(...args)`